pages/about.html

```diff
@@ -0,0 +1,447 @@
+              fill="black"
+            />
+          </g>
+          <defs>
+            <clipPath id="clip0_3_2">
+              <rect width="801" height="461" fill="white" />
+            </clipPath>
+          </defs>
+        </svg>
+
+        <figcaption class="about-hero-text-3">
+          The logo of KI LLC was decided through the logo competition from 8.
+          June to 7. July.
+        </figcaption>
+      </figure>
+    </main>
+
+    <section class="about-past">
+      <h2 class="about-past-heading">Product Launch</h2>
+      <p class="about-past-text">
+        Take a look at the last two KI Luncheons which took place at our
+        headquarter in Del Boco Vista
+      </p>
+      <ul class="mt-20" id="productslaunch-ul"></ul>
+    </section>
+
```

Comment:
- [ ] I think you are missing a section here. If you check at the design, you will find two images showcasing previous events, so regardless of the topic you chose for your project, it is a requirement for you to include all the sections given in the template, so please make sure to follow it.
Here's is an image of what I think you are missing in your about page:
![Captura de pantalla 2022-03-10 211539](https://user-images.githubusercontent.com/59100477/157747512-bc453a9e-51b5-4132-9301-e826f74e89c0.jpg)